applications/asset_tracker/README.rst

```diff
@@ -155,6 +155,10 @@ For example, configure ``CONFIG_POWER_OPTIMIZATION_ENABLE`` to enable power opti
 In |SES|, select **Project** > **Configure nRF Connect SDK project** to browse and configure these options.
 Alternatively, use the command line tool ``menuconfig`` or configure the options directly in ``prj.conf``.
 
+By default, this application does **not** use the full :ref:`ug_bootloader`.
+The final merged application binary by default will boot directly into MCUboot.
+You can use the ``CONFIG_SECURE_BOOT`` option to enable the secure bootloader chain.
+
```

Comment:
```suggestion
Secure Boot
========

This application supports :ref:`ug_bootloader`.
To enable Secure Boot, set ``CONFIG_SECURE_BOOT=y``.
```

Reply:
Great idea. I've pushed something halfway in between :).